addons/crossword/src/presenter.js

```diff
@@ -98,6 +99,55 @@ function Addoncrossword_create(){
         }
     };
 
+    presenter.prepareCorrectAnswers = function() {
```

Comment:
Nie chciała by ta funkcja być otestowana? W przyszłości wprowadzenie zmian i czytanie kodu może nie być takie oczywiste, a jakby takie testy były, to jest czarno na białym czego od tej funkcji oczekujemy :)